packages/E2ETest/wdio/pages/LoginPage.ts

```diff
@@ -1,59 +0,0 @@
-/**
- * Copyright (c) Microsoft Corporation.
- * Licensed under the MIT License.
- */
-
-import { BasePage, by } from './BasePage';
-import {
-  USERNAME_ON_LOGIN,
-  PASSWORD_ON_LOGIN,
-  SUBMIT_ON_LOGIN,
-  LOGINRESULT_ON_LOGIN,
-  SHOWPASSWORD_ON_LOGIN,
-} from '@react-native-windows/tester/js/examples-win/LegacyTests/Consts';
-
-class LoginPage extends BasePage {
```

Comment:
>class LoginPage extends BasePage [](start = 0, length = 32)

I don't know if it's more reasonable to keep the page structure. With the number of test cases grows, the Helpers may become hard to maintain

[https://github.com/microsoft/react-native-windows/pull/6636/commits/e0920d5b3f6bfce9de1d0dfb469e5b94d4eacaa8](https://github.com/microsoft/react-native-windows/pull/6636/commits/e0920d5b3f6bfce9de1d0dfb469e5b94d4eacaa8)

Reply:
Page functionality was moved inline to the page specific tests, where Helpers contains a combination of the functionality in BasePage and HomePage, that all tests would want to use.

The pages and tests were already tightly coupled, and the pages didn't encapsulate state, so I found it be be easier to understand and faster to iterate on without the extra layer of indirection.